config.go

```diff
@@ -21,6 +21,7 @@ type Config struct {
 	MetricMaxLength     int       `yaml:"metric_max_length"`
 	NumReaders          int       `yaml:"num_readers"`
 	NumWorkers          int       `yaml:"num_workers"`
+	OmitEmptyHostname   bool      `yaml:"omit_empty_hostname"`
```

Comment:
Absolutely; Added. I don't think this is an "important" enough change to note anywhere else. I *could* add some discussion to "Lack of Host Tags for Aggregated Metrics", but seems like it would be extra confusing for very little purpose. People who care will look at either `example.yaml` or the Configuration section about the hostname, so they should find this as well.